File: LUA/Cap08/c08ex09.lua
-- c08ex09.lua

function fibonacci(n)
  if (n == 0) then
    return 0
  end
  if (n == 1) then
    return 1
  end
  if (n >= 2) then
    return fibonacci(n - 1) + fibonacci(n - 2)
  end
end

for i = 0, 14 do
  print(string.format("%2d - %d", i, fibonacci(i)))
end

io.write("\nTecle <Enter> para encerrar... ")
io.read()

File: PYTHON/Cap08/c08ex09.py
# c08ex09.py

def fibonacci(n):
  if (n == 0):
    return 0
  if (n == 1):
    return 1
  if (n >= 2):
    return fibonacci(n - 1) + fibonacci(n - 2)

for i in range(15):
  print(f"{i:2} - {fibonacci(i)}")

input("\nPressione <Enter> para encerrar... ")
